Podfile

```diff
@@ -5,6 +5,8 @@ yajlVersion='~> 2.0.4'
 
 target 'AdblockPlusSafari' do
 
+pod 'yajl', $yajlVersion
```

Comment:
No, it is triggered by new Cocoa pods dependency, which was injected into main Xcode target. Almost all of those changes is this one line related.